bin/swan-configure.js

```diff
@@ -15,11 +15,8 @@ const questions   = {
     message : 'What is your github username:',
     default : (config.git) ? config.git.user.name : '',
     validate: function (input) {
-      if (input && input.length >= 5) {
-        return true;
-      }
-
-      return `Expected a valid github username.`;
+      let valid = typeof input === 'string' && input.length > 0;
```

Comment:
Mixing return types isn't ideal. I'd prefer it the way it was (conditional) but I won't decline the PR if you don't.